storage/time-series/influxdb/retriever/app/express/routes/v1/Devices.js

```diff
@@ -109,5 +126,67 @@ module.exports = ({ mountPoint, queryDataByField, queryDataByMeasurement }) => {
     ],
   };
 
-  return [deviceRoute, deviceAttrRoute];
+
+  /**
+   * This endpoint returns data fetched using graphql schema
+   */
+  const deviceGraphqlRoute = {
```

Comment:
This endpoint should be in another file, destined for graphql. This file is for `/devices/`

Reply:
You retrieve device information using graphql. But, we could split it into 2 files, one Http and graphQl.

Reply:
The folder is called "routes"

Reply:
It seems reasonable to add /devices/ to the endpoint, so I just updated accordingly.